app/design/frontend/base/default/template/somethingdigital/criteo/success.phtml

```diff
@@ -1,20 +1,40 @@
 <?php
     echo $this->getChildHtml('somethingdigital_criteo_device_detection');
+    $email = Mage::helper('somethingdigital_criteo')->getHashedEmail();
+    $orderId = $this->getData('order_id');
+    $items = $this->getCriteoOrderItems();
+    $useDataLayer = Mage::getStoreConfig('somethingdigital_criteo/settings/use_datalayer');
 ?>
 
-<script type="text/javascript">
-    window.criteo_q = window.criteo_q || [];
-    window.criteo_q.push(
-        { event: "setAccount", account: "<?php echo Mage::getStoreConfig('somethingdigital_criteo/settings/account_number'); ?>" },
-        { event: "setHashedEmail", email: ["<?php echo Mage::helper('somethingdigital_criteo')->getHashedEmail();?>"] },
-        { event: "setSiteType", type: deviceType },
-        { event: "trackTransaction",
-            <?php if (Mage::getStoreConfigFlag('somethingdigital_criteo/settings/use_deduplication')): ?>
-                deduplication: (Mage.Cookies.get('criteo_deduplication')) ? 1 : 0,
-            <?php endif; ?>
-            id: "<?php echo $this->getData('order_id'); ?>",
-            new_customer: <?php echo $this->getCriteoIsNewCustomer(); ?>,
-            item: <?php echo $this->getCriteoOrderItems(); ?>
-        }
-    );
-</script>
+<?php if (!$useDataLayer): ?>
+    <script type="text/javascript">
+        window.criteo_q = window.criteo_q || [];
+        window.criteo_q.push(
+            { event: "setAccount", account: "<?php echo Mage::getStoreConfig('somethingdigital_criteo/settings/account_number'); ?>" },
+            { event: "setHashedEmail", email: ["<?php echo Mage::helper('somethingdigital_criteo')->getHashedEmail();?>"] },
+            { event: "setSiteType", type: deviceType },
+            { event: "trackTransaction",
+                <?php if (Mage::getStoreConfigFlag('somethingdigital_criteo/settings/use_deduplication')): ?>
+                    deduplication: (Mage.Cookies.get('criteo_deduplication')) ? 1 : 0,
+                <?php endif; ?>
+                id: "<?php echo $orderId ?>",
+                new_customer: <?php echo $this->getCriteoIsNewCustomer(); ?>,
```

Comment:
Do we not want to pass this?

Reply:
By the guide this is not something that is required http://guides.criteotilt.com/tag-manager/gtm/?accountid=52244&advertiserid=47350&advertisername=Baked+by+Melissa&utm_medium=52244&homepageurl=http%3A%2F%2Fwww.bakedbymelissa.com%2F&accuratekey=bakedbymelissaus%3B7815412%3B7815413%3B52244#sales